### PR TITLE
Add Swift Package Index badges

### PR DESCRIPTION
**Pull request by calda**
This PR adds Swift Package Index badges that link to our package's [SPI listing](https://swiftpackageindex.com/airbnb/swift). 

I figured we could specifically add them to the section that discusses the package plugin. Alternatively we could put them at the very top of the page under the title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Note that brevity is not a primary goal. Code should be made more concise only i
 
 ## Swift Package Manager command plugin
 
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fairbnb%2Fswift%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/airbnb/swift) [![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fairbnb%2Fswift%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/airbnb/swift)
+
```
Comment (fdiaz) [inline README.md:27]: Should we put this at the top of this README? That's what I've usually seen in other repos.

**Review by fdiaz — APPROVED** [review]
Love it! Just one small comment.

File: README.md (hunk 1 above)
Comment (calda) [reply README.md:27]: Sounds good, that works too

Comment (calda) [reply README.md]: https://github.com/airbnb/swift/pull/178/commits/c25074138a38e8f6228f701ca04665ff9fd38a14

**Pull request merged**